File: covid-19/processing/src/main/scala/bean/CovidData.scala
package bean

import java.sql.Date

/**
 * @Author jiang.weiyu
 * @Date 2021/01/30 19:04
 */
case class CovidData(
                      // date
                      date: Date,
                      // province
                      name_jp: String,
                      // patients
                      npatients: Int,

                    )

File: covid-19/datasource/src/main/java/com/lagou/bean/CovidData.java
package com.lagou.bean;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.sql.Date;

/**
 * @Author jiang.weiyu
 * @Date 2021/01/30 1:39
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
public class CovidData {
    // date
    private Date date;
    // province
    private String name_jp;
    // patients
    private String npatients;
}
